projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -629,3 +631,5 @@ export function isSmartFolder(context: RuleContext): boolean {
 export const areTagsEnabled = (context: AcaRuleContext): boolean => context.appConfig.get('plugins.tagsEnabled', true);
 
 export const areCategoriesEnabled = (context: AcaRuleContext): boolean => context.appConfig.get('plugins.categoriesEnabled', true);
+
+export const isNodeFrozen = (_context: RuleContext, node: NodeEntry): boolean => !!node?.entry?.properties?.['rma:frozenAt'];
```

Comment:
This aspect should only be available from governance, this rule should not be placed in ACA